src/api/json/catalog.json

```diff
@@ -3159,6 +3159,12 @@
       "fileMatch": ["templatesources.json"],
       "url": "https://json.schemastore.org/templatesources.json"
     },
+    {
+      "name": "Tier.run pricing.json",
+      "description": "Tier.run pricing model configuration",
+      "fileMatch": ["pricing.json"],
```

Comment:
The file match seems a bit generic. Does that file always go in a folder with a particular name or is there something you can do to make it less generic?

Reply:
It doesn't really have to be in any particular folder, no. Even naming it `pricing.json` is really just a convention. https://docs.tier.run/docs/learning-tier/getting-started-with-tier/pricing-json